test/clj/rems/api/test_applications.clj

```diff
@@ -683,17 +683,39 @@
         api-key "42"
         wf-id (test-data/create-workflow! {:type :workflow/default
                                            :handlers [handler]})
-        form-id (test-data/create-form! {})
-        cat-id (test-data/create-catalogue-item! {:workflow-id wf-id
+        form-id (test-data/create-form! {:form/fields [{:field/id "fld1"
+                                                        :field/type :text
+                                                        :field/title {:en "Field 1"}
+                                                        :field/optional false}]})
+        form-2-id (test-data/create-form! {:form/fields [{:field/id "fld2"
+                                                          :field/type :text
+                                                          :field/title {:en "HIDDEN"}
+                                                          :field/optional false}]})
+        cat-id (test-data/create-catalogue-item! {:title {:en "Item1"}
+                                                  :workflow-id wf-id
                                                   :form-id form-id})
-        app-id (test-data/create-application! {:actor applicant
-                                               :catalogue-item-ids [cat-id]})]
+        cat-2-id (test-data/create-catalogue-item! {:title {:en "Item2"}
+                                                    :workflow-id wf-id
+                                                    :form-id form-2-id})
+        app-id (test-data/create-draft! applicant [cat-id] "Answer1")
+        app-2-id (test-data/create-draft! applicant [cat-id cat-2-id] "Answer2")]
     (send-command applicant {:type :application.command/submit
                              :application-id app-id})
+    (send-command applicant {:type :application.command/submit
+                             :application-id app-2-id})
     (testing "reporter can export"
       (let [exported (api-call :get (str "/api/applications/export?form-id=" form-id) nil
-                               api-key reporter)]
-        (is (= (count (str/split exported #"\n")) 2))))
+                               api-key reporter)
+            [header & lines] (str/split-lines exported)]
+        (is (.contains exported "Field 1")
```

Comment:
possibly `str/includes?`

Reply:
I was looking for str/contains? and didn't find it...